buildstockbatch/test/test_docker_base.py

```diff
@@ -27,20 +27,23 @@ def test_prep_batches(basic_residential_project_file, mocker):
     dbb.batch_array_size = 3
     DockerBatchBase.validate_project = MagicMock(return_value=True)
 
-    with tempfile.TemporaryDirectory(prefix="bsb_") as tmpdir, tempfile.TemporaryDirectory(
-        prefix="bsb_"
-    ) as tmp_weather_dir:
-        dbb._weather_dir = tmp_weather_dir
+    with tempfile.TemporaryDirectory(prefix="bsb_") as tmpdir:
         tmppath = pathlib.Path(tmpdir)
-
-        job_count, unique_epws = dbb.prep_batches(tmppath)
+        epws_to_copy, n_sims, job_count = dbb._prep_batch_files(tmppath)
         sampler_mock.run_sampling.assert_called_once()
 
-        # Of the three test weather files, two are identical
-        assert sorted([sorted(i) for i in unique_epws.values()]) == [
-            ["G2500210.epw"],
-            ["G2601210.epw", "G2601390.epw"],
-        ]
+        # There are three weather files...
+        #   * "G2500210.epw" is unique; check for it (gz'd) in tmppath
+        #   * "G2601210.epw" and "G2601390.epw" are dupes. One should be in
+        #     tmppath; one should be copied to the other according to ``epws_to_copy``
+        assert os.path.isfile(tmppath / "weather" / "G2500210.epw.gz")
+        assert os.path.isfile(tmppath / "weather" / "G2601210.epw.gz") or os.path.isfile(
+            tmppath / "weather" / "G2601390.epw.gz"
+        )
+        src, dest = epws_to_copy[0]
+        assert src == "G2601210.epw.gz" or src == "G2601390.epw.gz"
+        assert dest == "G2601210.epw.gz" or dest == "G2601390.epw.gz"
```

Comment:
```suggestion
        assert src in ("G2601210.epw.gz", "G2601390.epw.gz")
        assert dest in ("G2601210.epw.gz", "G2601390.epw.gz")
```
nit: this feels more Pythonic to me